src/indexer/services/contract/index.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Bridge as BridgeType,
+  BasicFeeHandler as BasicFeeHandlerType,
+  FeeHandlerWithOracle as FeeHandlerWithOracleType,
+  ERC20Handler as ERC20HandlerType,
+  ERC721Handler as ERC721HandlerType,
+  GenericHandler as GenericHandlerType,
```

Comment:
@nmlinaric we replace here `GenericHandler` for  `PermissionlessGenericHandler`?

@MakMuftic are we going to include `PermissionedGenericHandler`?

Reply:
`GenericHandler` was renamed to `PermissionedGenericHandler`, `PermissionlessGenericHandler` is the new contract that will be used.
We won't be deploying `PermissionedGenericHandler`.